gateway/api/utils.py

```diff
@@ -421,13 +421,10 @@ def create_dependency_allowlist():
 
 def sanitize_name(name: str | None):
     """Sanitize name"""
-    if name:
-        sanitized_name = ""
-        for c in name:
-            if c.isalnum() or c in ["_", "-", "/"]:
-                sanitized_name += c
-        return sanitized_name
-    return name
+    if not name:
+        return name
+    # Remove all characters except alphanumeric, _, -, /
+    return re.sub("[^a-zA-Z0-9_\\-/]", "", name)
```

Comment:
```suggestion
    return re.sub("[^a-zA-Z0-9_-/]", "", name)
```
I think you don't need the backslash.

Reply:
It's not mandatory but it's recommended. In practice, for this specific use case, both will likely work the same way. However, the escaped version \- is considered a best practice. It explicitly tells the regex parser to treat the hyphen as a literal character and avoids potential ambiguity with the character interpretation.

Reply:
Ok, I didn't know about it, then you can replicate in the other regex. Thank you for the information